src/Main.cc

```diff
@@ -333,6 +333,57 @@ static int tclReadlineInit(Tcl_Interp* interp)
 }
 #endif
 
+namespace {
+// A fallback from the hardcoded TCLRL_LIBRARY path.
+std::string findPathToTclreadlineInit(Tcl_Interp* interp)
+{
+  std::string path(TCLRL_LIBRARY "/tclreadlineInit.tcl");
+  if (is_regular_file(path.c_str())) {
+    return path;
+  }
+
+  // TL;DR it is possible to run the OpenROAD binary from within the
+  // official Docker image on a different distribution than the
+  // distribution within the Docker image.
+  //
+  // In this case we have to look up
+  // the location of the tclreadline scripts instead of using the hardcoded
+  // path.
+  //
+  // It is helpful to use the official Docker image as CI infrastructure and
+  // also because it is a good way to have as similar an environment as possible
+  // during testing and deployment.
+  //
+  // See
+  // https://github.com/The-OpenROAD-Project/bazel-orfs/blob/main/docker.BUILD.bazel
+  // for the details on how this is done.
```

Comment:
I don't see any details there that explain how it is done.  Its just a list of files and there is no mention of docker.

Reply:
Documentation on **how** this is done belongs in bazel-orfs. Admittedly it could be better.